test/integration.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const assert = require('assert');
+const blc = require('broken-link-checker');
+const test = require('selenium-webdriver/testing');
+const webdriver = require('selenium-webdriver');
+
+const timeOut = 35000;
+
+const baseURL = process.env.VS_TEST_URL || 'https://viewsourceconf-stage.us-west.moz.works';
+console.log(`Running integration tests against ${baseURL}`)
+
+describe('The site...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+
+    it('should not have any broken links', function(done) {
+        const options = {
+            // https://github.com/stevenvachon/broken-link-checker
+            excludedKeywords: ['/2015', 'livereload'],
+            filterLevel: '3',
+        };
+
+        const errors = {};
+
+        const siteChecker = new blc.SiteChecker(options, {
+            link(result) {
+                if (result.broken) {
+                    errors[result.base.resolved] = errors[result.base.resolved] || [];
+                    errors[result.base.resolved].push(
+                        `${result.url.resolved} (${blc[result.brokenReason]})`
+                    );
+                }
+            },
+            site() {
+                if (Object.keys(errors).length > 0) {
+                    const msg = `Site has broken links: \n ${JSON.stringify(errors)}`;
+                    return done(new Error(msg));
+                }
+                return done();
+            },
+        });
+
+        siteChecker.enqueue(url, url);
+    });
+});
+
+test.describe('The landing page...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+    const expectedTitle = 'View Source Conference 2016';
+
+    test.it(`should be called "${expectedTitle}"`, function(done) {
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
+            .build();
+
+        driver.get(url).then(function() {
+            driver.getTitle().then(function(title) {
+                assert.equal(title, expectedTitle)
+            });
+            done();
+        });
+    });
+
+    test.it('should have a footer & footer should have a link', function(done) {
+        const selector = 'footer ul .footer_social_twitter a';
+        const expectedText = '@ViewSourceConf';
+
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
+            .build();
+
+        driver.get(url).then(function() {
+            driver.findElement(webdriver.By.css(selector)).then(function(link) {
+                link.getText().then(function(text) {
+                    assert.equal(text.toLowerCase(), expectedText.toLowerCase());
```

Comment:
Asserting things like the text of a button/title/link isn't really all that useful (imho). It makes for a brittle test (all you need to do is change the copy for it to fail), and only really asserts that you didn't change the HTML (which you will likely already know if you have?). 

Would it be more useful to assert that these elements are displayed instead? For example, a CSS regression could cause an element to be hidden for some reason, but the test here as it stands could still pass?